src/controllers/director.confirm.correspondence.address.controller.ts

```diff
@@ -47,5 +48,18 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
 };
 
 export const post = async (req: Request, res: Response, next: NextFunction) => {
-  return res.redirect(urlUtils.getUrlToPath(DIRECTOR_RESIDENTIAL_ADDRESS_PATH, req));
+  try {
+    const transactionId = urlUtils.getTransactionIdFromRequestParams(req);
+    const submissionId = urlUtils.getSubmissionIdFromRequestParams(req);
+    const session: Session = req.session as Session;
+
+    // Patch filing with updated information
```

Comment:
this comment should not mention updated information. Say reset or clear or just remove comment

Reply:
Comments used across project but I am glad to remove it, method name is self explanatory

Removed